components/ui/Banner/index.tsx

```diff
@@ -10,36 +10,25 @@ export default function Banner(): ReactElement {
   return (
     <div
       className={classNames(
-        'bg-gray-dark text-white py-4 px-8 flex flex-col justify-center items-center leading-relaxed',
-        'lg:flex-row lg:items-start',
+        'bg-gray-dark text-white py-4 px-8 flex flex-col justify-center items-center leading-relaxed align-middle',
+        'lg:flex-row',
         '2xl:items-center'
       )}
     >
-      <span className={classNames('text-center mb-4', 'lg:text-left lg:mb-0')}>
+      <span className={classNames('text-center', 'lg:text-left lg:mb-0')}>
```

Comment:
we probably need the mb-4 back because things are a little tight on mobile.

<img width="451" alt="Screenshot 2024-04-02 at 14 46 51" src="https://github.com/oxen-io/session-website/assets/14887287/71eb5a81-f4e5-44cf-89ca-2f5e236090c4">

Reply:
Added `gap-2` to address this

## Mobile
![image](https://github.com/oxen-io/session-website/assets/5667907/eba93fa4-b15c-4449-9f49-81086e782eeb)


## Desktop
![image](https://github.com/oxen-io/session-website/assets/5667907/03ec9337-4c97-4962-9e11-493d46469588)